tests/integration.rs

```diff
@@ -5396,6 +5410,125 @@ mod tests {
                     .spawn()
                     .unwrap()
             }
+
+            // XXX Follow up test involving multiple disks will require:
+            // - Make image size variable
+            // - Make image filename random
+            // - Cleanup image file after test
+            // - Use NTFS instead of FAT, needs mkfs.ntsf in the container
+            fn disk_new(&self) -> String {
+                let img = PathBuf::from(
+                    String::from_utf8_lossy(b"/tmp/test-ntsf-hotplug-0.raw").to_string(),
+                );
+                let _ = fs::remove_file(&img);
+
+                // Create an image file
+                let out = Command::new("qemu-img")
+                    .args(&["create", "-f", "raw", &img.to_str().unwrap(), "100m"])
+                    .output()
+                    .expect("qemu-img command failed")
+                    .stdout;
+                println!("{:?}", out);
+
+                // Associate image to a loop device
+                let out = Command::new("losetup")
```

Comment:
Run `losetup` with `--show`, it will return the device that was picked.

Reply:
Yeah, that simplifies a lot. Thanks for the hint!